### PR TITLE
Feature more options

### PR DESCRIPTION
**Pull request by j-licht**
Added options fps, width and height.

### CODE REVIEW

File: fuse-ts.c [hunk 1]
```diff
@@ -33,11 +33,14 @@ char *start_time = NULL;
 char *prefix = NULL;
 int inframe = 0;
 int outframe = 0;
+int frames_per_second = 50;
```
Comment (a-tze) [inline fuse-ts.c]: This must be 25 for several reasons.

Comment (j-licht) [reply fuse-ts.c]: OK. We can take 25 as default.

Comment (a-tze) [reply fuse-ts.c]: YOU did choose 25 as the default ;)

See 890789da85031f6c14080f45aeb0a3e32e48f871

Comment (j-licht) [reply fuse-ts.c]: I see, then it was a "real" miss teak.

**Comment by a-tze** [comment]
Ok just curios: did you test this with NTSC-style fps values, e.g. 59.97? Just want to document if that works flawlessly with KDEnlive etc.

**Comment by j-licht** [comment]
No. I think then I have to change some code, because now fps is a int and 59.97 is a float or a double.

Do you think this is feature is required?

**Comment by a-tze** [comment]
Actually... no. Anyone who needs this can set it to 60, that doesnt hurt for this use case.

**Pull request merged**